docs/text-style-props.md

```diff
@@ -405,9 +405,29 @@ Set to `false` to remove extra font padding intended to make space for certain a
 
 ### `fontVariant`
 
-| Type                                                                                                       | Default |
-| ---------------------------------------------------------------------------------------------------------- | ------- |
-| array of enum(`'small-caps'`, `'oldstyle-nums'`, `'lining-nums'`, `'tabular-nums'`, `'proportional-nums'`) | `[]`    |
+| Type | Default |
+| ---- | ------- |
+
+| array of enum(`'small-caps'`, `'oldstyle-nums'`, `'lining-nums'`, `'tabular-nums'`, `'proportional-nums' `,`'stylistic-one'`,
+, `'stylistic-two'`
+, `'stylistic-three'`
+, `'stylistic-four'`
+, `'stylistic-five'`
+, `'stylistic-six'`
+, `'stylistic-seven'`
+, `'stylistic-eight'`
+, `'stylistic-nine'`
+, `'stylistic-ten'`
+, `'stylistic-eleven'`
+, `'stylistic-twelve'`
+, `'stylistic-thirteen'`
+, `'stylistic-fourteen'`
+, `'stylistic-fifteen'`
+, `'stylistic-sixteen'`
+, `'stylistic-seventeen'`
+, `'stylistic-eighteen'`
+, `'stylistic-nineteen'`
+, `'stylistic-twenty'`) | `[]` |
```

Comment:
```suggestion
| array of enum(`'small-caps'`, `'oldstyle-nums'`, `'lining-nums'`, `'tabular-nums'`, `'proportional-nums' `,`'stylistic-one'`, `'stylistic-two'`, `'stylistic-three'`, `'stylistic-four'`, `'stylistic-five'`, `'stylistic-six'`, `'stylistic-seven'`, `'stylistic-eight'`, `'stylistic-nine'`, `'stylistic-ten'`, `'stylistic-eleven'`, `'stylistic-twelve'`, `'stylistic-thirteen'`, `'stylistic-fourteen'`, `'stylistic-fifteen'`, `'stylistic-sixteen'`, `'stylistic-seventeen'`, `'stylistic-eighteen'`, `'stylistic-nineteen'`, `'stylistic-twenty'`) | `[]` |
```

Unfortunately, making table row multiline breaks the MD renderer.

<img width="659" alt="Screenshot 2022-09-06 112431" src="https://user-images.githubusercontent.com/719641/188599084-5522db25-1e3c-4ef6-97b8-8ee280078078.png">

Reply:
Done, Thank you for the feedback @Simek

Reply:
The table still appears broken after the deploy with the suggested changes, so it looks like I need to checkout out it locally to looks at it and fix it. 😅 Maybe you would like to give it a try @gedeagas ?

Reply:
wait, let me check again. sorry for the inconvenience

Reply:
Hmm, it looks like the table is still displaying incorrectly. 🤔

Reply:
Hi sorry, i havent push the fix yet. will do later today